builder/make_get_image_dependencies

```diff
@@ -48,11 +48,27 @@ for feature in "${features[@]}"; do
 		script="features/$feature/convert$extension"
 		input="$artifact_base-$COMMIT.raw"
 	fi
+
+	shopt -s nullglob
+	set +f
```

Comment:
Maybe add comment here what the `-f` does and why it's necessary. I had to look it up, since I do not recall all parameter from `set`.

Reply:
Thanks